### ISSUE TITLE
Support for M1 ARM?

### ISSUE THREAD

**Issue by Strat1987** [kmikiy/SpotMenu#262]
Are there plans to support M1 ARM native?

**Comment by kmikiy** [comment]
yes, closing because I'm rewriting the app from scratch https://github.com/kmikiy/SpotMenu/issues/279#issuecomment-2987565341
